compiler/plugins/input/Torch/torch-mlir/CMakeLists.txt

```diff
@@ -116,10 +116,15 @@ iree_cc_library(
   DEPS
     ::defs
     ::TorchConversionDialectGen
+    ::TorchDialectTransformsGen
     ::TorchDialectIR
     MLIRIR
+    MLIRFuncTransforms
+    MLIRGPUDialect
```

Comment:
Why do we have a dependency on GPU and Vector in torch now?

Reply:
The gcc CI broke on missing `iree::compiler::Dialect::Stream::IR`. I ran the missing dependencies check and fixed all the ones that came up.

Reply:
The torch-mlir bump had to make the same change: https://github.com/iree-org/iree/commit/58691e7c4f59acd3a1a0714945e0feb71db97e8d#diff-431fd5242ff55680ab08967de6608c763884259ac621350fbd709b360086ae94R58

Reply:
I see, thanks for the pointer. Seems like an unusual dependency so probably should look at why Torch-MLIR took those deps then.